tests/integration-tests/ganache-reverts/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "ganache-reverts",
+  "version": "0.1.0",
+  "scripts": {
+    "build-contracts": "rm -rf abis bin && solcjs contracts/Contract.sol --abi -o abis && mv abis/*Contract.abi abis/Contract.abi && solcjs contracts/Contract.sol --bin -o bin && mv bin/*Contract.bin bin/Contract.bin",
+    "codegen": "graph codegen",
+    "test": "yarn build-contracts && truffle test --network test",
+    "create:test": "graph create test/ganache-reverts --node http://localhost:18020/",
+    "deploy:test": "graph deploy test/ganache-reverts --ipfs http://localhost:15001/ --node http://localhost:18020/"
+  },
+  "devDependencies": {
+    "@graphprotocol/graph-cli": "https://github.com/graphprotocol/graph-cli#master",
```

Comment:
can you try that with graph-cli 0.17.0 and graph-ts 0.17.1

Reply:
As commented offline, `graph test` was only added after graph-cli 0.17.1, so no.